pb_plugins/dcsdkgen/utils.py

```diff
@@ -55,6 +60,12 @@ def extract_string_type(field):
         return "UNKNOWN_TYPE"
 
 
+def filter_out_result(fields):
+    """ Filters out the result fields (".*Result$") """
+    for field in fields:
+        if not extract_string_type(field).endswith("Result"): yield field
```

Comment:
- Styling; yield in new line
- Add e.g. `is_result(..)` which contains `extract_string_type(field).endswith("Result")`